src/en/developer-guide/applications/utilities/create-and-open.md

```diff
@@ -0,0 +1,28 @@
+---
+author: Lukáš Gallovič
+---
+
+# Create And Open - Entity Generator
+
+### This is a tutorial on how to use custom TS code to generate an entity from a different entity.
+You have to add `JS function` to RibbonDiff.xml, so it looks something like this.
+You must use `createAndOpen` function and `talxis_entitygenerator.js` library.
+```
+<JavaScriptFunction FunctionName="createAndOpen" Library="$webresource:talxis_entitygenerator.js">
+<CrmParameter Value="PrimaryControl" />
+<StringParameter Value="ait_GenerateCase" /> (name of my action)
+</JavaScriptFunction>
+```
+U must use `2 parameters` the first one is always PrimaryControl and the second one is the name of your workflow action.
```

Comment:
U -> You.

This is public facing material... It can be funny, but it should be grammatically correct.